client/js/reactComponents.jsx

```diff
@@ -132,8 +132,8 @@ class ChatMessage extends React.Component {
 	render() {
 		var username = dataManager.users[this.props.data.sender].username;
 		return (
-			<div style={{"backgroundColor": "rgba(255, 255, 255, 0.5)", "marginTop":"2px"}}>
-				<span style={{"fontSize": "70%", "color":"white"}}>{username}<br/></span>
```

Comment:
(`.message > span` is a terrible selector, both from a performance POV ([it will slow down all `span`s on the page](https://stackoverflow.com/questions/5797014/), admittedly not by very much in this case), and from an understanding/maintainability perspective. Give the span a reasonable class instead, it's easy enough to change later.)